src/plugins/controls/common/control_group/control_group_panel_diff_system.ts

```diff
@@ -74,4 +75,29 @@ export const ControlPanelDiffSystems: {
       );
     },
   },
+  [TIME_SLIDER_CONTROL]: {
+    getPanelIsEqual: (initialInput, newInput) => {
+      if (!deepEqual(omit(initialInput, 'explicitInput'), omit(newInput, 'explicitInput'))) {
+        return false;
+      }
+
+      const {
+        isAnchored: isAnchoredA,
+        timesliceStartAsPercentageOfTimeRange: startA,
+        timesliceEndAsPercentageOfTimeRange: endA,
+      }: Partial<TimeSliderControlEmbeddableInput> = initialInput.explicitInput;
+      const {
+        isAnchored: isAnchoredB,
+        timesliceStartAsPercentageOfTimeRange: startB,
+        timesliceEndAsPercentageOfTimeRange: endB,
+      }: Partial<TimeSliderControlEmbeddableInput> = newInput.explicitInput;
+      return (
+        Boolean(isAnchoredA) === Boolean(isAnchoredB) &&
+        Boolean(startA) === Boolean(startB) &&
+        startA === startB &&
+        Boolean(endA) === Boolean(endB) &&
+        endA === endB
+      );
```

Comment:
This fixes a bug where resetting the selections back to "undefined" / empty would cause unsaved changes:

**Before:**

**After:**